### PR TITLE
📃 missing file from a link in README.md

### PR DESCRIPTION
**Pull request by CatScan-crypt**
but its still messing the correct value for starkPrivateKey
if i send it like this node returns 

ReferenceError: starkPrivKey is not defined.

if i declare  "const starkPrivKey = ethPrivKey"   i get

"error":"ETH_STARKKEY_NOT_MATCHED" .

opened an issue about this.

i'm using infura on a local node, balance and history are ok

### CODE REVIEW

File: examples/node_sell_eth_infura.js [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+  const validFor = '0'
+  const feeRate = ''
+
+
+  const submitOrderResponse = await dvf.submitOrder({
+
+    symbol,
+    amount,
+    price,
+    starkPrivateKey: starkPrivKey,
```
Comment (AliMeer) [inline examples/node_sell_eth_infura.js:41]: starkPrivKey isn't defined.

**Comment by AliMeer** [comment]
Please elaborate on what this PR is meant to add?

File: examples/node_sell_eth_infura.js (hunk 1 above)
Comment (CatScan-crypt) [reply examples/node_sell_eth_infura.js:41]: the file is fine just needs to remove res.send in 52 
and add  const starkPrivKey = ethPrivKey

File: examples/node_sell_eth_infura.js [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+    feeRate,            // Optional
+    gid: '1',           // Optional
+    cid: '1',           // Optional
+    partnerId: 'P1'    // Optional
+  })
+
+  console.log('submitOrder response ->', submitOrderResponse)
+
+
+    res.send(submitOrderResponse);
```
Comment (CatScan-crypt) [inline examples/node_sell_eth_infura.js:52]: remove this

File: examples/node_sell_eth_infura.js [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+
+
+const DVF = require('dvf-client-js')
+
+
+const privateKey = '8F085...' // Account's private key
+const infuraKey = '9e28b...'  // Your Infura API KEY
+const infuraURL = 'https://mainnet.infura.io/v3/' + infuraKey
+
+
```
Comment (CatScan-crypt) [inline examples/node_sell_eth_infura.js:15]: this here
const starkPrivKey = privateKey

File: examples/node_sell_eth_infura.js [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+const HDWalletProvider = require('truffle-hdwallet-provider')
+const sw = require('starkware_crypto')
+const Web3 = require('web3')
+
+
+const DVF = require('dvf-client-js')
+
+
+const privateKey = '8F085...' // Account's private key
```
Comment (CatScan-crypt) [inline examples/node_sell_eth_infura.js:11]: replace privateKey
with
const ethPrivKey

**Comment by AliMeer** [comment]
@CatScan-crypt The PR adds this file `examples/node_sell_eth_infura.js` However, this file requires a number of corrections so this cant be merged as it is.

**Comment by CatScan-crypt** [comment]
should i close this and open a new tested version? 
thanks to you i can make this happen  😁

**Comment by AliMeer** [comment]
Closing this one, welcome to create a new PR with further suggested improvements.

**Pull request closed without merging**